### ISSUE TITLE
Investigate undiscovered tasks

### ISSUE THREAD

**Issue by jaredpalmer** [vercel/turborepo#28]
From Lattice: ```Apologies if this has been asked a bunch of times (I’ve looked). Every time I run turbo I get 0 cached, 0 total. I’ve tried re-installing node, rim raffing node_modules etc to no avail. Other folks at my org have had this problem and others don’t.```

**Comment by promer94** [comment]
I just came across the similar problem
 
<img width="690" alt="Screen Shot 2021-12-09 at 23 46 55" src="https://user-images.githubusercontent.com/15878786/145428863-db6aa453-fcae-4fc7-a8d9-ec16d3075602.png">

And it works with single star.
<img width="690" alt="Screen Shot 2021-12-10 at 00 04 45" src="https://user-images.githubusercontent.com/15878786/145432096-97b15fe7-404b-45bc-b8e0-48e8ed92388f.png">

When using double star globs, it seems all the node_module build script will be executed

<img width="690" alt="Screen Shot 2021-12-09 at 23 46 55" src="https://user-images.githubusercontent.com/15878786/145432497-06fbf6e5-4fd7-4e1c-bccb-839f76032e08.png">

**Comment by jaredpalmer** [comment]
Doublestar bug: #32

**Comment by revelt** [comment]
My current monorepo uses npm and I didn't set up the workspaces, I used to use `lerna bootstrap` because I was jumping between Node 14 and Node 16. The former is where `tap` unit test coverage works, but only the latter supports workspaces.

Anyway, until I set up npm workspaces, I was getting `0 cached, 0 total` too! Turbo was ending in 100ms. For context, my 120 packages build on Lerna with `--concurrency 1` takes around an hour.

Whoever is on npm and getting `0 total`, ensure you've got workspaces set up, use `engines` to enforce Node 16:

```diff
// root package.json
+  "workspaces": [
+.   "packages/*"
+  ],
+  "engines": {
+    "node": ">=16.0.0"
+  },
```
